holidays/countries/united_states.py

```diff
@@ -208,7 +231,31 @@ def _populate_subdiv_holidays(self):
             "VI",
         }:
             name = "Washington's Birthday"
-            if self._year >= 1971:
+            if self._year >= 1971 and self.subdiv not in {
+                "AK",
+                "CA",
+                "CO",
+                "HI",
+                "ID",
+                "MD",
+                "MN",
+                "MT",
+                "NJ",
+                "OH",
+                "OK",
+                "OR",
+                "PA",
+                "PR",
+                "SC",
+                "TN",
+                "TX",
+                "UT",
+                "VA",
+                "VT",
+                "WA",
+                "WV",
+                "WY",
```

Comment:
I think these lines should be moved to condition above (L222).

Reply:
The reason I did it the way I did is to prevent removing “Washington's Birthday” from each subdivision pre-1971. By moving these lines up to the condition above, it removes the holiday before the rename, which is I don't think is correct.